files/dashboards/openshift-logging-dashboard.json

```diff
@@ -7,8 +7,8 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
-  "id": null,
-  "iteration": 1594321382217,
+  "id": 1,
+  "iteration": 1616139020699,
```

Comment:
not sure why id and iteration was changed .... you might want to get those back to original if not needed

Reply:
reverted.